docs/shared/android/configure_java.md

```diff
@@ -38,27 +36,9 @@ To see what version of Java you have installed, run:
 java -version
 ```
 
-If you have Java 8 configured, you'll see output like:
-```
-openjdk version "1.8.0_242-release"
-OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
-OpenJDK 64-Bit Server VM (build 25.242-b3-6222593, mixed mode)
-```
-
-### Error running pre-push hooks
-If you encounter this error, particularly when running our recommended pre-push hooks:
+If you have Java 17 configured, you'll see output like:
 ```
-Could not initialize class org.codehaus.groovy.runtime.InvokerHelper
+openjdk 17.0.7 2023-04-18
+OpenJDK Runtime Environment Homebrew (build 17.0.7+0)
+OpenJDK 64-Bit Server VM Homebrew (build 17.0.7+0, mixed mode, sharing)
 ```
-
-You may be on Java 14: consider installing Java 8 (see above). If preferred, other users have found installing any version of Java, 13 or lower, appears to resolve the issue:
```

Comment:
Looking into this, it looks like it might not be the case anymore.

Reply:
I don't think this is the case any more. 🤔 If the [git blame here](https://github.com/mozilla-mobile/shared-docs) shows this is old enough, maybe we're better of just killing this bit.

Reply:
the info is from 3 years ago.